lib/Backend/Vault/Kv.hs

```diff
@@ -303,7 +303,7 @@ kvDeleteSecret backend path = do
 instance Backend VaultKvBackend where
   _name kvBackend = vbName kvBackend
   _codec = vaultKvCodec
-  _writeSecret = kvWriteSecret
-  _readSecret = kvReadSecret
-  _listSecrets = kvListSecrets
-  _deleteSecret = kvDeleteSecret
+  _writeEntry = kvWriteSecret
+  _readEntry = kvReadSecret
+  _listEntries = kvListSecrets
+  _deleteEntry = kvDeleteSecret
```

Comment:
Please, rename these functions (`kvWriteSecret` and etc)

Reply:
Done. There are corresponding functions in `Vault.Internal`, but I'm not sure if I need to change those.

Reply:
I think we want to unify these namings. So we should perform `secret` -> `entry` renaming

Reply:
Done

Reply:
Makes sense